append.c

```diff
@@ -54,7 +54,7 @@
 /*
  * Internal method to process a long or int number and write into a buffer.
  */
-static	char	*handle_decimal(char *buf, char *limit, const long num, const int base)
+static	char	*handle_decimal(char *buf, char *limit, const long long num, const int base)
```

Comment:
Decimal is either long or int.  I don't support %ll or %ql  right now.  Since I'm printing pointers as %p I don't think this should be there.

Reply:
I don't want to force support for long long either.  It's not supported on some architectures.

Reply:
Okay, keeping support for pre-C99 platforms in an entirely valid goal!